frontend/src/pages/Account.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import '../styles/App.scss';
+
+class AccountPage extends Component {
+  render() {
+    return (
+      <div class="container">
+        <div class="row">
+          <div class="col-md-9">
+            <h4>My Account</h4>
+            <br />
+            <form
+              class="form-horizontal"
+              role="form"
+              method="post"
```

Comment:
Our form does not need a method, as it will use React calls to perform the put request and handle the response.